src/modules/GameConstants.ts

```diff
@@ -360,6 +360,16 @@ export enum Currency {
     contestToken,
 }
 
+export enum CurrencyRate {
```

Comment:
Your names are a bit generic. If someone just saw `CurrencyRate` in GameConstants, they would not be able to guess what it's used for

Reply:
CurrencyExchangeRate maybe?

Reply:
The values are only used for luxury balls, right? Or are you planning to use the values for something else?

Reply:
I am not planning on using them for anything else

Reply:
Then include Luxury ball in the name, so other coders knows that it's what the numbers are balanced for

Reply:
Done, should be more clear now